packages/react/src/defaultTheme.ts

```diff
@@ -0,0 +1,124 @@
+import { Theme } from "./index";
```

Comment:
nice that this file is extracted! super clear. Maybe rename to defaultThemes or create separate files for light / dark